vispy/scene/cameras/base_camera.py

```diff
@@ -349,16 +365,24 @@ def link(self, camera):
         ----------
         camera : instance of Camera
             The other camera to link.
+        axis : str of aligning axis := "x" / "y"
+            syncs only with one axis between cameras
         """
-        cam1, cam2 = self, camera
-        # Remove if already linked
-        while cam1 in cam2._linked_cameras:
-            cam2._linked_cameras.remove(cam1)
-        while cam2 in cam1._linked_cameras:
-            cam1._linked_cameras.remove(cam2)
-        # Link both ways
-        cam1._linked_cameras.append(cam2)
-        cam2._linked_cameras.append(cam1)
+        if axis is not None:
+            main_cam, sec_cam = self, camera
+            main_cam._linked_cameras.append([sec_cam, axis])
+            sec_cam._linked_cameras.append([main_cam, axis])
+            self.master = True
+        else:
+            cam1, cam2 = self, camera
+            # Remove if already linked
+            while cam1 in cam2._linked_cameras:
+                cam2._linked_cameras.remove(cam1)
+            while cam2 in cam1._linked_cameras:
+                cam1._linked_cameras.remove(cam2)
+            # Link both ways
+            cam1._linked_cameras.append(cam2)
+            cam2._linked_cameras.append(cam1)
```

Comment:
Can we combine the `axis` None and not None cases? Or make it so linked cameras *always* has either `"x"` or `"y"` or `None`?

Reply:
Yes i suppose, since None is default it shouldn't break the Code of people who use the old implementations of link